src/modules/UI/components/FlipInput/CryptoExchangeFlipInputWrapperComponent.js

```diff
@@ -78,9 +78,10 @@ export class CryptoExchangeFlipInputWrapperComponent extends Component<Props, St
             </View>
         </View>
     }
-    const {guiWallet: {name: guiWalletName, currencyCode}} = this.props
+    const {guiWallet: {name: guiWalletName},
```

Comment:
This syntax is rather confusing and hard to read. Please break this up into several lines that more clearly dereference `this.props`